dbt_project.yml

```diff
@@ -23,14 +23,14 @@ vars:
     subscription: "{{ source('recharge','subscription') }}"
     subscription_history: "{{ source('recharge','subscription_history') }}"
 
-    recharge__address_passthrough_columns: []
-    recharge__charge_passthrough_columns: []
-    recharge__charge_line_item_passthrough_columns: []
-    recharge__order_passthrough_columns: []
-    recharge__order_line_passthrough_columns: []
-    recharge__product_passthrough_columns: []
-    recharge__subscription_passthrough_columns: []
-    recharge__subscription_history_passthrough_columns: []
+    recharge__address_passthrough_columns: [] 
+    recharge__charge_passthrough_columns: [] 
+    recharge__charge_line_item_passthrough_columns: [] 
+    recharge__order_passthrough_columns: [] 
+    recharge__order_line_passthrough_columns: [] 
+    recharge__product_passthrough_columns: [] 
+    recharge__subscription_passthrough_columns: [] 
+    recharge__subscription_history_passthrough_columns: [] 
```

Comment:
I am having some difficulty getting this to function without the empty variables. I think this is because the variable name is passed to `fill_pass_through_columns` as a string instead of the var contents.

Do you know of any packages that I can reference? I was searching through our repo to try to find an example, or most recent packages and am seeing they use the empty brackets in dbt_project.yml.

Reply:
Oh also--is this something maybe that could be added to [fivetran_utils](https://github.com/fivetran/dbt_fivetran_utils/blob/releases/v0.4.latest/macros/fill_pass_through_columns.sql)? Having the default be that empty list instead of having no default on line 4?

Reply:
Having discussed internally, we do in fact still need to keep these default declarations in the package dbt_project.yml.

I think it would be great to open a feature request for this default variable declaration to be part of the macro.

Reply:
Added back in, and issue submitted!